src/sagemaker/workflow/README.rst

```diff
@@ -0,0 +1,162 @@
+=============================
+SageMaker Workflow in Airflow
+=============================
+
+Apache Airflow
+~~~~~~~~~~~~~~
+
+`Apache Airflow <https://airflow.apache.org/index.html>`_
+is a platform that enables you to programmatically author, schedule, and monitor workflows. Using Airflow,
+you can build a workflow for SageMaker training, hyperparameter tuning, batch transform and endpoint deployment.
+You can use any SageMaker deep learning framework or Amazon algorithms to perform above operations in Airflow.
+
+There are two ways to build SageMaker workflow. Using Airflow SageMaker operators or using Airflow PythonOperator.
+
+1. SageMaker Operators: Since Airflow 1.10.1, we contributed special operators just for SageMaker operations.
+Each operator takes a configuration dictionary that defines the corresponding operation. And we provide APIs to
+generate the configuration dictionary in SageMaker Python SDK. Current supported SageMaker operators are:
+
+* ``SageMakerTrainingOperator``
+* ``SageMakerTuningOperator``
+* ``SageMakerModelOperator``
+* ``SageMakerTransformOperator``
+* ``SageMakerEndpointConfigOperator``
+* ``SageMakerEndpointOperator``
+
+2. PythonOperator: Airflow built-in operator that could execute Python callables. You could use SageMaker Python SDK to
+customize your workflow with PythonOperator.
+
+Using Airflow on AWS
+~~~~~~~~~~~~~~~~~~~~
+
+Turbine is an open source AWS CloudFormation template to create Airflow resources stack on AWS.
+You can get it here: https://github.com/villasv/aws-airflow-stack
+
+Using Airflow SageMaker Operators
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Starting from Airflow 1.10.1, you could use SageMaker operators in Airflow. All SageMaker operators take a
```

Comment:
Starting wiht Airflow 1.10.1, you can use SageMaker operators in Airflow. All SageMaker operators take a configuration dictionary that can be generated by the SageMaker Python SDK. For example:

Reply:
Updated.